rpm-build/SPECS/securedrop-workstation-dom0-config.spec

```diff
@@ -43,16 +43,14 @@ install -m 755 -d %{buildroot}/srv/salt/sd/sd-workstation
 install -m 755 -d %{buildroot}/srv/salt/sd/sys-firewall
 install -m 755 -d %{buildroot}/usr/share/%{name}/scripts
 install -m 755 -d %{buildroot}/srv/salt/sd/usb-autoattach
+install -m 755 -d %{buildroot}/%{_bindir}
```

Comment:
This is necessary as the directory was previously created as part of the `setup.py` run.

Reply:
(As an aside,  `%{_bindir}` has a leading slash, so those sorts of invocations resolve to `//`. That's a noop, but can omit the unnecessary `/` - which is also used in the original code - if that's preferred.)